makefile

```diff
@@ -35,6 +43,36 @@ release: | tmp/output/release/
 	@test $(VERSION)
 	gh release create $(VERSION) --notes "New release." --title "$(VERSION)" tmp/output/release/*
 
+# Mappings
+# todo: I'd like to do USE_PREBUILT=false, but this error is occuring:
+#  cp: cannot stat 'tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv': No such file or directory
+#  ...even though I checked and the file at that path does exist.
+USE_PREBUILT=true
+
+tmp/input/mondo/:
+	rm -rf $@ &&\
+	cd tmp/input/ &&\
+	git clone --depth 1 https://github.com/monarch-initiative/mondo
+
+tmp/input/mondo.sssom.tsv: tmp/input/mondo/
+	if [ $(USE_PREBUILT) = true ]; then wget https://raw.githubusercontent.com/monarch-initiative/mondo/master/src/ontology/mappings/mondo.sssom.tsv -O $@; else cd tmp/input/ &&\
+		cd mondo/src/ontology &&\
+		make mondo.owl mappings -B MIR=false IMP=false MIR=false &&\
+		cp tmp/input/mondo/src/ontology/mappings/mondo.sssom.tsv $@; fi
+
+# todo: Stable URI/filename issue: https://github.com/monarch-initiative/icd11/pull/12#discussion_r1542187711
+tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml: | tmp/input/
+	wget https://www.orphadata.com/data/nomenclature/packs/Orphanet_Nomenclature_Pack_EN.zip -O tmp/input/Orphanet_Nomenclature_Pack_EN.zip
+	unzip tmp/input/Orphanet_Nomenclature_Pack_EN.zip -d tmp/input/Orphanet_Nomenclature_Pack_EN
+
+tmp/output/release/mondo-icd11.sssom.tsv tmp/output/release/ordo-icd11.sssom.tsv: tmp/input/Orphanet_Nomenclature_Pack_EN/ORPHA_ICD11_mapping_en_newversion_2023.xml tmp/input/mondo.sssom.tsv | tmp/output/release/
```

Comment:
## `mondo-icd11.sssom.tsv`
- [ ] @sabrinatoro Review
I created a [google sheet](https://docs.google.com/spreadsheets/d/1npe1PdZrb0YTtAbPqgu7OZ961g7KwC0_eZLAlK7RCEc/edit#gid=1894001781). Please review!

@matentzn @twhetzel FYI
I created this by first getting the ORDO->ICD11 mappings, then doing a join on exact mappings from `mondo.sssom.tsv`.

Reply:
This has been reviewed, and is now utilized post mondo-ingest build.